test/format/test_columns.py

```diff
@@ -0,0 +1,13 @@
+import pip_audit.format as format
+
+from .vuln_data import TEST_VULN_DATA
+
+
+def test_columns():
+    columns_format = format.ColumnsFormat()
+    expected_columns = """Package Version ID     Description              Affected Versions
+------- ------- ------ ------------------------ ------------------------------------
+foo     1.0     VULN-0 The first vulnerability  (0.9 => fix: 1.1), (N/A => fix: 1.4)
```

Comment:
Wasn't sure how to best represent version ranges in a columnar format. Let me know if you have ideas!

Reply:
This seems reasonable to me (I think anything we do is going to be a bit shoehorned, since we're mapping a list of range into a single cell).